File: app/src/main/java/sgtmelon/handynotes/model/state/DragState.java
package sgtmelon.handynotes.model.state;

import sgtmelon.handynotes.interfaces.ItemClick;

public class DragState implements ItemClick.Drag {

    private boolean drag = false;

    public boolean isDrag() {
        return drag;
    }

    @Override
    public void setDrag(boolean itemDrag) {
        this.drag = itemDrag;
    }
}

File: app/src/main/java/sgtmelon/scriptum/office/state/DragState.kt
package sgtmelon.scriptum.office.state

import sgtmelon.scriptum.office.intf.ItemIntf

/**
 * Состояние для перетаскивания элементов, определяющее можно перетаскивать его или нет
 */
class DragState : ItemIntf.DragListener { // TODO (убрать)

    private var drag = false

    fun isDrag(): Boolean {
        return drag
    }

    override fun setDrag(drag: Boolean) {
        this.drag = drag
    }

}